BP/common/governments/00_government_types.txt

```diff
@@ -187,11 +182,12 @@ holy_order_government = {
 	ai = {
 		arrange_marriage = no
 		use_goals = no
+		use_scripted_guis = no
 		perform_religious_reformation = no
 	}
 
 	flag = can_not_be_vassal_or_liege
 
 	valid_holdings = { castle_holding }
 	color = hsv{ 0.00 0.00 0.66 }
-}
+}
```

Comment:
Why do you modify this in this pull request? Because of the patch?

Reply:
Yeah